### PR TITLE
Fix npx issue

### PR DESCRIPTION
**Pull request by anthmatic**
GH has started using a newer version of node/npm where there are breaking changes to the behavior of the `npx` command

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -20,13 +20,15 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Install NPM deps
```
Comment (anthmatic) [inline action.yml:23]: We don't need to install any deps

File: action.yml [hunk 2]
```diff
@@ -20,13 +20,15 @@ inputs:
...
-      run: npm install
-      shell: bash
+    - uses: actions/checkout@v2
```
Comment (anthmatic) [inline action.yml:23]: We're now more explicit about the setup for this and pinning node to version 16.

File: action.yml [hunk 3]
```diff
@@ -20,13 +20,15 @@ inputs:
...
+      uses: actions/setup-node@v1
+      with:
+        node-version: '16.x'
     - name: Deploy
       env:
         HUBSPOT_PORTAL_ID: ${{ inputs.portal_id }}
         HUBSPOT_PERSONAL_ACCESS_KEY: ${{ inputs.personal_access_key }}
       run: |
-        npx @hubspot/cli@3.0.9 upload ${{ inputs.src_dir }} ${{ inputs.dest_dir }} --use-env 
+        npx --yes --package=@hubspot/cli@3.0.9 --call='hs upload ${{ inputs.src_dir }} ${{ inputs.dest_dir }} --use-env'
```
Comment (anthmatic) [inline action.yml:33]: - NPX now prompts the user to install the package, `--yes` skips that
- `--package` explicitly installs the latest version
- `--call` runs the actual CLI command

**Review by brandenrodgers — APPROVED** [review]
👌

**Comment by anthmatic** [comment]
Fixes #15

**Pull request merged**